cinema-frontend/src/app/model/movie.ts

```diff
@@ -10,9 +10,19 @@ export interface Movie {
 }
 
 export enum MovieGenre {
```

Comment:
Consider that you will add new movie genre – then you should add new one in backend and frontend, imo better will be download list of all genres from backend